packages/webcomponents/src/components/business-forms/owner-form/owner-form.tsx

```diff
@@ -0,0 +1,243 @@
+    if (!this.authToken) console.error(missingAuthTokenMessage);
+
+    this.formController = new FormController(ownerSchema);
+    this.api = Api(this.authToken, config.proxyApiOrigin);
+    this.fetchData();
+  }
+
+  componentDidLoad() {
+    this.formController.errors.subscribe(
+      errors => (this.errors = { ...errors }),
+    );
+    this.formController.values.subscribe(
+      values => (this.owner = { ...values }),
+    );
+  }
+
+  inputHandler = (name: string, value: string) => {
+    this.formController.setValues({
+      ...this.formController.values.getValue(),
+      [name]: value,
+    });
+  }
+
+  onAddressFormUpdate = (values: any): void => {
+    this.formController.setValues({
+      ...this.formController.values.getValue(),
+      address: {
+        ...this.formController.values.getValue().address,
+        ...values,
+      }
+    });
+  }
+  
+  private validateAndSubmit(event: any) {
+    event.preventDefault();
+    this.formController.validateAndSubmit(this.sendData);
+  }
+
+  render() {
+
+    const ownerDefaultValue =
+      this.formController.getInitialValues();
+
+    return (
+      <Host exportparts="label,input,input-invalid">
+        <form onSubmit={this.validateAndSubmit}>
+          <fieldset>
+            <legend>{this.formTitle}</legend>
+            <hr />
+            <div class="row gy-3">
+              <div class="col-12 col-md-8">
+                <form-control-text
+                  name="name"
+                  label="Full Name"
+                  defaultValue={ownerDefaultValue?.name}
+                  error={this.errors.name}
+                  inputHandler={this.inputHandler}
+                />
+              </div>
+              <div class="col-12 col-md-4">
+                <form-control-select
+                  name="title"
+                  label="Prefix"
+                  defaultValue={ownerDefaultValue?.title}
+                  options={[
+                    { label: 'Select Prefix', value: '' },
+                    { label: 'Mrs.', value: 'Mrs.' },
+                  ]}
+                  error={this.errors.title}
+                  inputHandler={this.inputHandler}
+                />
+              </div>
+              <div class="col-12 col-md-6">
+                <form-control-text
+                  name="email"
+                  label="Email Address"
+                  defaultValue={ownerDefaultValue?.email}
+                  error={this.errors.email}
+                  inputHandler={this.inputHandler}
+                />
+              </div>
+              <div class="col-12 col-md-6">
+                <form-control-number-masked
+                  name="phone"
+                  label="Phone Number"
+                  defaultValue={ownerDefaultValue?.phone}
+                  error={this.errors.phone}
+                  inputHandler={this.inputHandler}
+                  mask={PHONE_MASKS.US}
```

Comment:
Nice, I didn't realize we had a masking mechanism! I was looking at the schema file, and phone is only `required`. I know we're going to do some work on the validation requirements for this form later, but I wonder if we should make the validation schema for this field match whatever the mask is expecting... thoughts?

Reply:
Good call! 

I went ahead and added some additional validation to the phone part of the schema for this component. We can add it to the forms that require phone numbers after the dust settles.